src/cxy_ctl.erl

```diff
@@ -58,14 +61,43 @@ make_process_dictionary_default_value(Key, Value) ->
 
 %% Use a raw tuple because two record structs with the same key would
 %% be needed, or redundant record name in key position.
+
+%% Cxy counters tuple...
 -define(MAX_PROCS_POS,    2).
 -define(ACTIVE_PROCS_POS, 3).
 -define(MAX_HISTORY_POS,  4).
 
-make_proc_values(Task_Type, Max_Procs_Allowed, Max_History) ->
+%% Cumulative performance moving average tuple...
+-define(CMA_TYPE_POS,        1).
+-define(CMA_SPAWN_CMA_POS,   2).
+-define(CMA_EXEC_CMA_POS,    3).
+-define(CMA_RING_SIZE_POS,   4).
+-define(CMA_SLOW_FACTOR_POS, 5).
```

Comment:
is there any gain from this - other than obfuscating the code? A record would give you all of this for free. and it would be way easier to read.